queue-staging.tf

```diff
@@ -0,0 +1,75 @@
+module "envelopes-staging-queue" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-queue?ref=master"
+  name                = "envelopes"
+  namespace_name      = "${module.queue-namespace.name}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
+
+  requires_duplicate_detection            = "true"
+  duplicate_detection_history_time_window = "PT59M"
+  lock_duration                           = "PT5M"
+  max_delivery_count                      = "${var.envelope_queue_max_delivery_count}"
+}
+
+module "processed-envelopes-staging-queue" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-queue?ref=master"
+  name                = "processed-envelopes"
```

Comment:
Same as for the envelopes queue. Is it an exact duplicate of the non-staging queue?